docs/gen_ref_pages.py

```diff
@@ -0,0 +1,25 @@
+"""Generate the code reference pages."""
+
+from pathlib import Path
+
+import mkdocs_gen_files
+
+for path in sorted(Path("src").rglob("*.py")):  # 
```

Comment:
Is there a reason for the empty comments at the end of each line? I noticed this is also failing pre-commit due to the extra whitespace at the end

Reply:
Fixing it in template